bitnami/prometheus-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.41.0
 description: The Prometheus Operator for Kubernetes provides easy monitoring definitions for Kubernetes services and deployment and management of Prometheus instances.
 name: prometheus-operator
-version: 0.32.0
+version: 0.32.1
```

Comment:
In this case I think we should increase at least to a "minor" version (i.e. 0.33.0) as it adds a few feature.